lib/Pod/Simple/Wiki/Confluence.pm

```diff
@@ -103,7 +103,9 @@ sub _handle_text {
     my @tokens = split /(\s+)/, $text;
 
     # Escape any tokens here, if necessary.
-    @tokens = map { s/([[{*_-])/\\$1/g; $_ } @tokens;
+    unless( $self->{_in_Verbatim} ) {
```

Comment:
As a general rule I only use `unless` as a postfix equality. I think I got that from Perl Best Practices. So use this instead:

```perl
if ( ! $self->{_in_Verbatim} ) {
```
Also, this seems to fix issue #20. Could you verify that case as well.